src/tcgwars/logic/impl/gen3/FireRedLeafGreen.groovy

```diff
@@ -2523,11 +2523,11 @@ public enum FireRedLeafGreen implements LogicCardInfo {
               checkLastTurn()
               powerUsed()
               new Knockout(self).run(bg)
```

Comment:
You could go with this approach with the name but you'd have to modify it to say `!it.name.contains(" ex ")` 

Another approach is just to change it to check for the Pokemon SubType by using `!it.topPokemonCard.cardTypes.is(EX)`